packages/flame_sprite_fusion/example/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: example
```

Comment:
Name it `<<package>>_example` otherwise it could collide with other example names in the future

Reply:
Done 👍🏻